app/src/main/kotlin/it/vfsfitvnm/vimusic/ui/components/themed/MediaItemMenu.kt

```diff
@@ -222,7 +251,6 @@ fun BaseMediaItemMenu(
     MediaItemMenu(
         mediaItem = mediaItem,
         onDismiss = onDismiss,
-        onGoToEqualizer = onGoToEqualizer,
```

Comment:
there is still `fun BaseMediaItemMenu(... onGoToEqualizer: (() -> Unit)? = null ...)` parameter which is unused. Please delete it as well